### ISSUE TITLE
Add "console" for easier execution of SQL queries

### ISSUE THREAD

**Issue by Sobak** [Albert221/FromSelect#1]
PMA has something like this but I think it can be more promoted. I would go with some *aggressive* key binding like `Ctrl+Q` so it would be really useful. Not as a full replacement for "SQL" tab but in case of simpler queries it could reduce workflow by one click (opening new tab/page reload).

**Comment by Albert221** [comment]
Yeah, absolutely! I thought about small popup being shown after pressing some key binding which would include no more than a simple textarea for SQL query.

**Comment by Sobak** [comment]
Popup could look nice but it obviously has shlightly reduced functionality. You can't show history of queries.

That's not the case with typical `SELECT` (as it would reload the page anyway) but things like `SELECT COUNT` or other returning single result would benefit from classic console-like approach.

**Comment by Albert221** [comment]
Hmm, that approach would be really clever. That's definitely a case and I'm looking forward to implement this!
